cmd/oras/manifest/fetch.go

```diff
@@ -113,18 +114,18 @@ func fetchManifest(opts fetchOptions) (fetchErr error) {
 		// fetch manifest descriptor only
 		fetchOpts := oras.DefaultResolveOptions
 		fetchOpts.TargetPlatform = opts.Platform.Platform
-		desc, err = oras.Resolve(ctx, src, opts.RawReference, fetchOpts)
+		desc, err = oras.Resolve(ctx, src, opts.Reference, fetchOpts)
 		if err != nil {
-			return err
+			return fmt.Errorf("failed to find %q in %q: %w", opts.Reference, opts.RawReference, err)
```

Comment:
What will this error message look like? It seems that it contains duplicated messages.